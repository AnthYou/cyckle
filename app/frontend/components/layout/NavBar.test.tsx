import React from "react";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";

import NavBar from "./Navbar";

describe("<NavBar />", () => {
  it("should render correctly", () => {
    const component = renderer.create(
      <BrowserRouter>
        <NavBar />
      </BrowserRouter>
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
