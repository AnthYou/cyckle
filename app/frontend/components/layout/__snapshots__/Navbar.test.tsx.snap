// Vitest Snapshot v1

exports[`<Navbar /> > should render correctly 1`] = `
<nav
  className="container flex flex-wrap items-center"
>
  <h3>
    <a
      href="/"
      onClick={[Function]}
      style={
        {
          "color": "#373737",
          "fontFamily": "Playfair-Display",
        }
      }
    >
      Cyckle
    </a>
  </h3>
  <ul
    className="ml-auto flex space-x-4"
  >
    <li
      className="flex flex-col justify-center items-center"
    >
      <a
        className="text-gray-600 hover:text-sky-600 font-medium undefined"
        href="/bikes"
        onClick={[Function]}
      >
        Trouver un v√©lo
      </a>
    </li>
    <li
      className="flex flex-col justify-center items-center"
    >
      <a
        href="/login"
        onClick={[Function]}
      >
        <button
          className="_Button_4ea4ec _alternate_4ea4ec"
        >
          S'inscrire / se connecter
        </button>
      </a>
    </li>
  </ul>
</nav>
`;
